### DISCUSSION TITLE
The house of the dead 2 error in textures

### DISCUSSION THREAD

**Discussion by 3lc3z4r** [flyinghead/flycast#651] [Q&A]
Any solution? I have tried to change the graphical Api on both devices. The game runs normal for me at 60 fps in both cases I even increased the native resolution to x2. I am using flycast emulator in both cases. images 1-2(pc),3-
![imagen_2022-05-30_150100395](https://user-images.githubusercontent.com/106557341/171053602-197e27b0-1932-4b67-bdd6-9479b572f4db.png)
![imagen_2022-05-30_150132711](https://user-images.githubusercontent.com/106557341/171053643-709a4f95-7022-4b84-9fc3-c2e9baba3b8a.png)
4(android)

![imagen_2022-05-30_150402658](https://user-images.githubusercontent.com/106557341/171053854-ec78b83d-7ef6-483f-96d2-3977c26076a2.png)
![imagen_2022-05-30_150413748](https://user-images.githubusercontent.com/106557341/171053873-5f6782de-9170-47ae-acbf-7c46ba627dc0.png)

**Comment by flyinghead** [comment]
See https://github.com/flyinghead/flycast/issues/570
